#left_robot:
joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50

wheel_controller_fr:
  type: effort_controllers/JointVelocityController
  joint: front_right_wheel_joint
  pid: { p: 10.0, i: 0.1, d: 0.1 }

wheel_controller_fl:
  type: effort_controllers/JointVelocityController
  joint: front_left_wheel_joint
  pid: { p: 10.0, i: 0.1, d: 0.1 }

wheel_controller_rr:
  type: effort_controllers/JointVelocityController
  joint: rear_right_wheel_joint
  pid: { p: 10.0, i: 0.1, d: 0.1 }

wheel_controller_rl:
  type: effort_controllers/JointVelocityController
  joint: rear_left_wheel_joint
  pid: { p: 10.0, i: 0.1, d: 0.1 }